packages/health-check/src/checks/verbose.js

```diff
@@ -1086,7 +1087,7 @@ function hnsEndpointDirectoryRedirect(done) {
     },
   };
 
-  skylinkVerification(done, expected, { redirects: 0 });
+  skylinkVerification(done, expected, { followRedirect: false });
```

Comment:
This function is now `async`. Shouldn't we `await` here (and everywhere else)?

Reply:
yeah we might but not in this iteration yet, we would have to refactor all 40ish test functions